### PR TITLE
Add API docs for getElementOrNull

### PR DESCRIPTION
**Pull request by jkrems**
Docs for https://github.com/testiumjs/testium-driver-sync/pull/7

### CODE REVIEW

**Comment by khoomeister** [comment]
:shipit:

**Comment by jkrems** [comment]
Published in `testium-driver-sync@2.2.0`.

File: API.md [hunk 1]
```diff
@@ -138,6 +138,18 @@ button = browser.getElement('.button')
...
 
+### browser.getElementOrNull(cssSelector)
+
+Finds an element on the page
+using the `cssSelector`
+and returns an [Element](#element).
+Returns `null` if the element wasn't found.
+
+```coffee
+button = browser.getElement('.button')
```
Comment (EndangeredMassa) [inline API.md]: this is the wrong method; should be `getElementOrNull`, right?

Comment (jkrems) [inline API.md]: Yes, it should be. Fixing. Thanks!

**Pull request merged**